CompanyEmployees.Presentation/Controllers/EmployeesController.cs

```diff
@@ -22,11 +23,22 @@ public IActionResult GetEmployeesForCompany(Guid companyId)
             return Ok(employees);
         }
 
-        [HttpGet("{id:guid}")]
+        [HttpGet("{id:guid}", Name = "GetEmployeeForCompany")]
```

Comment:
You can just name this method "Get employee"

Reply:
i have renamed GetEmployeeForCompany to GetEmployee